py/makeqstrdefs.py

```diff
@@ -15,14 +15,10 @@
 QSTRING_BLACK_LIST = {'NULL', 'number_of', }
 
 
-def debug(message):
-    #sys.stderr.write("%s\n" % message)
-    pass
-
-
 def write_out(fname, output):
     if output:
-        fname = fname.replace("/", "__").replace("..", "@@")
+        for c in ["/", "\\", ":", ".."]:
+            fname = fname.replace(c, "")
```

Comment:
It might be important to replace these characters with something non-empty, to avoid clashing file names.  Easy to do:
```
for from, to in (('/', '__'), ('..', '@@'), ...):
    ...
```